Documentation/Books/Manual/ReleaseNotes/KnownIssues33.md

```diff
@@ -0,0 +1,13 @@
+Known Issues
+============
+
+This page lists some known issues of the ArangoDB suite of products. 
```

Comment:
Maybe make this text a bit more useful and clarify what this is and what it isn't?
```suggestion
This page lists important issues of the ArangoDB suite of products all users
should take note of. It is not a list of all open issues (see
[GitHub Issues](https://github.com/arangodb/arangodb/issues) for that).
```

Reply:
thanks - the problem with adding the link "https://github.com/arangodb/arangodb/issues" is that it is a partial list as other issues are in other repositories